api/routes.py

```diff
@@ -152,4 +153,23 @@ def put_to_toolhub(tool, data):
   header = {"Authorization": f'Bearer {flask.session["token"]["access_token"]}'}
   response = requests.put(url, data=data, headers=header)
   r = response.status_code
-  return r
\ No newline at end of file
+  return r
+
+user = Blueprint("user", __name__, description="Get information about the currently logged-in user.")
+
+@user.route("/api/user")
+class CurrentUser(MethodView):
+  @tasks.response(200, UserSchema)
+  def get(self):
```

Comment:
You could reuse the `get_current_user` function defined higher up here.

Reply:
Bizarrely enough, I had it in my mind to have the get_current_user make a call to the API endpoint... I am not sure why I thought that was a good idea.  Doing it this way around makes a lot more sense!